Gruntfile.js

```diff
@@ -8,7 +8,7 @@ module.exports = function(grunt) {
 
   gtx.config(gruntConfig);
 
-  gtx.alias('build', ['build-dist', 'less', 'copy']);
+  gtx.alias('build', ['build-dist', 'less', 'sass', 'copy']);
```

Comment:
The idea behind this was that i head that shortbreaks use SASS and therefore i put a compiler in for them to be able to make use of it easily.